DFC.App.ExploreCareers.Data/Contracts/ICosmosRepository.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Net;
+using System.Threading.Tasks;
+using DFC.App.ExploreCareers.Data.Models;
+using DFC.Compui.Cosmos.Contracts;
+
+namespace DFC.App.ExploreCareers.Data.Contracts
+{
+    public interface ICosmosRepository<T>
```

Comment:
You do not  need to define this interface yourself as it is defined in the Cosmos nuget